.gitignore

```diff
@@ -0,0 +1,3 @@
+.idea
+out
+java_barman_v2.iml
```

Comment:
Instead of writing full .iml file name you can use wildcard. So it will be *.iml instead of java_barman_v2.iml